projects/js-packages/connection/README.md

```diff
@@ -56,7 +56,7 @@ const statusCallback = useCallback(
 	apiRoot="https://example.org/wp-json/" 
 	apiNonce="12345"
 	registrationNonce="54321"
-	from="connection-ui"
+	from="my-jetpack"
```

Comment:
We should probably replace it with something generic so we wouldn't mess up our stats when somebody copies the code without replacing the `from` keys.
What do you think?

Reply:
I don't think so. This is real usage and people can grep the code for and see the real-world implementation. Coming up with an abstract name won't add much for a person's purpose of just making it work